qcodes/dataset/dond/sweeps.py

```diff
@@ -203,3 +206,40 @@ def num_points(self) -> int:
     @property
     def post_actions(self) -> ActionsT:
         return self._post_actions
+
+
+class MultiSweep:
+    """
+    A combination of Multiple sweeps that are to be performed in parallel
+    such that all parameters in the `MultiSweep` are set to the next value
+    before a parameter is read.
+
+    """
+
+    def __init__(self, sweeps: Sequence[AbstractSweep]):
```

Comment:
Could also be

```suggestion
    def __init__(self, *sweeps: AbstractSweep):
```

?

Reply:
``*sweeps`` is likely to be more appreciated by users working in scripts/notebooks for its convenience. And in a more "static" code doing ``MultiSweep(*my_list_of_sweeps)`` instead of ``MultiSweep(my_list_of_sweeps)`` is not a big deal.